OpenRA.Mods.Common/Activities/Resupply.cs

```diff
@@ -130,12 +124,8 @@ public override bool Tick(Actor self)
 			else if (activeResupplyTypes != 0 && aircraft == null && !isCloseEnough)
 			{
 				var targetCell = self.World.Map.CellContaining(host.Actor.CenterPosition);
-				QueueChild(move.MoveWithinRange(host, closeEnough, targetLineColor: moveInfo.GetTargetLineColor()));
 
-				// HACK: Repairable needs the actor to move to host center.
-				// TODO: Get rid of this or at least replace it with something less hacky.
```

Comment:
I'm not sure if we actually resolved these comments... There is still a `closeEnough` even for actors that should dock.

Reply:
It's resolved now that the default WDist is 512 instead of 0

Reply:
See my comments above, we shouldn't use 512 instead of 0.

Reply:
This won't be resolved in this PR. The use of 512 is a preexisting hack that fixes some cases but not all. The distance calculation itself will need a reimplementation. 

As it's a pretty large change requiring update rules so I won't be fixing in resupply code. It's going to be deleted anyway. Rather I'll just implement it properly in the new docking code / current refinery

Reply:
As we discussed with pchote the proper way is defining Cell offset and a WDist offset. 

Ground units need to be parked in a cell, so ground units would use the Cell offset. And Aircraft can be parked anywhere, so they will use the Cell + WDist offset. Ground units also have a way of using the WDist offset via drag, so that will also need to be accounted of